doc/workflows/anaconda-webui.rst

```diff
@@ -0,0 +1,156 @@
+Anaconda WebUI Testing Workflow
+===============================
+
+Workflow for running Anaconda WebUI integration tests.
+
+Requirements
+------------
+
+- Testing framework, taken from anaconda, cockpit, cockpit-project/bots git repositories. Configured in settings.
+- Test scripts, taken from anaconda or other git repository. Configured in settings.
+- QEMU Virtualization hypervisor/s. Configured in settings.
+- Event structure InstallationSourceStructure, this is what is going to be tested.
+  It can be obtained by conversion from compose structure or supplied directly::
+  
+    {
+        "base_repo_id": "BaseOS",
+        "repos": {
+            "BaseOS": {
+                "x86_64": {
+                    "os": "http://example.com/compose/x86_64/BaseOS/os",
+                    "kernel": "images/pxeboot/vmlinuz",
+                    "initrd": "images/pxeboot/initrd.img"
+                }
+            }
+        }
+    }
+
+
+Settings
+--------
+
+AnacondaWebUI
+^^^^^^^^^^^^^
+- **anaconda_repo** - URL to Anaconda git repository or path to local directory (use file://)
+- **cockpit_repo** - URL to Cockpit git repository, if anaconda_repo is local directory and
+  ``anaconda/ui/webui/test/common`` exists, this repo won't be used.
+- **cockpit_branch** - Cockpit git branch
+- **bots_repo** - URL to Cockpit Bots git repository, if anaconda_repo is local directory and
+  ``anaconda/ui/webui/bots`` exists, this repo won't be used.
+- **bots_branch** - Cockpit Bots git branch
+- **hypervisor_vm_limit** - How many VMs can this workflow use at once
+- **use_container** - Run npm command and the test itself inside podman container.
+- **port_ssh** - SSH port used for connection to the VM where Anaconda is running.
+- **port_webui** - Port used for connection to the Anaconda WebUI
+- **webui_location** - Part of URL where WebUI should be accessible.
+- **webui_startup_timeout** - Timeout in minutes for Anaconda WebUI to start after the VM gets IP
+- **webui_ssl_verify** - Verify SSL certificate used by Anaconda WebUI
+- **debug** - If true VMs is not removed at the end of the test
+
+VMHypervisors
+^^^^^^^^^^^^^
+Hypervisor hostnames or IPs, one for each supported architecture.
+Use `qemu:///system` for local hypervisor.
```

Comment:
How to define multiple hypervisors? Please, provide an example with hypervisors for multiple architectures.